pkg/vmconfig/volumes/unmanaged/util/unmanagedvolumes_util.go

```diff
@@ -58,12 +62,29 @@ type UnmanagedVolumeInfo struct {
 	Volumes map[string]vmopv1.VirtualMachineVolume
 }
 
+// FilterOutLinkedClones returns only the disks that are not linked clones.
+func FilterOutLinkedClones(disks ...VirtualDiskInfo) []VirtualDiskInfo {
+	return slices.DeleteFunc(
+		disks,
+		func(e VirtualDiskInfo) bool {
+			return e.LinkedClone
+		})
+}
+
+// FilterOutFCDs returns only the disks that are not FCDs.
+func FilterOutFCDs(disks ...VirtualDiskInfo) []VirtualDiskInfo {
+	return slices.DeleteFunc(
+		disks,
+		func(e VirtualDiskInfo) bool {
+			return e.FCD
+		})
+}
+
 // GetUnmanagedVolumeInfoFromVM returns information about a VM and its
 // unmanaged volumes.
 func GetUnmanagedVolumeInfoFromVM(
```

Comment:
It seems like it returns information about all disks and controllers on the VM, not just unmanaged ones?